v8/payloads/interactions.ts

```diff
@@ -84,7 +84,7 @@ export interface APIInteraction {
 	data?: APIApplicationCommandInteractionData;
 	guild_id: string;
 	channel_id: string;
-	member: APIGuildMember & { user: APIUser };
+	member: APIGuildMember & { permissions: string; user: APIUser };
```

Comment:
At this point might as well do `APIGuildMember & Required<Pick<APIGuildMember, 'user' | 'permissions'>>` so we can easily add extras on

Reply:
permissions isnt in GuildMember

Reply:
..oh
Well
This is why you don't do reviews at 7:42